.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         with:
           components: clippy
       - uses: Swatinem/rust-cache@v1
-      - run: cargo clippy --workspace --all-targets -- -D warnings
+      - run: cargo clippy --workspace --all-targets -- -D warnings -D rust_2018_idioms
```

Comment:
I would put this at the `crates/pathfinder/src/lib.rs` so there'd be more instant feedback.